library/general/validtags/template.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: templates.gatekeeper.sh/v1
+kind: ConstraintTemplate
+metadata:
+  name: k8svalidtags
+  annotations:
```

Comment:
can we add the new `metadata.gatekeeper.sh/title` annotation for the library website